fauna/faunadb-js
# Fix non-nullable `Select` default parameter type

### Notes

`null` is a valid value in FQL. Fixes #583.

### How to test

1. Pass `null` to the default parameter of [`Select`](https://docs.fauna.com/fauna/current/api/fql/functions/select?lang=javascript). No type error should be raised.

### Screenshots

N/A
